educator/src/Dscp/Educator/Web/Educator/Types.hs

```diff
@@ -120,6 +122,71 @@ requestToAssignment NewAssignment{..} =
     , _aDesc = naDesc
     }
 
+---------------------------------------------------------------------------
+-- Buildable instances
+---------------------------------------------------------------------------
+
+instance Buildable (NewCourse) where
+    build (NewCourse{..}) =
+      "{ course id = " +| ncId |+
+      ", description = " +| ncDesc |+
+      ", subjects = " +| fmap blockListF ncSubjects |+
```

Comment:
I think `listF` fits here better, otherwise if one passes many subjects (which are not that interesting in most cases), we would get an unreadable log.